docs/config-file.md

```diff
@@ -12,30 +12,84 @@ export default async function () {
 }
 ```
 
-This function must return an object with the following shape:
+This function must return an object with protocol specific configuration which would look like this: 
```

Comment:
I think we should leave it as it was. There's also the undocumented possibility to add the following configuration keys:

* GLEE_DIR
* GLEE_LIFECYCLE_DIR
* GLEE_FUNCTIONS_DIR
* ASYNCAPI_FILE_PATH

See https://github.com/asyncapi/glee/blob/master/src/lib/configs.ts#L41.

Also `cluster` is not a protocol but the configuration of the cluster mode.